src/blood-center/models/blood-center-event.entity.ts

```diff
@@ -0,0 +1,35 @@
+import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { BloodCenterEntity } from './blood-center.entity';
+
+@Entity('blood_centers_events')
```

Comment:
A `location/adress` table was proposed but idea was dropped. @matgrube might share some insight but overall I don't mind either way. If we decide for the `adress` to be separated it can be done now or later but before connecting the lambda preferably.

Reply:
Sure, I guess it's not _that bad_ yet. What about the postal code though?

Reply:
Addressed the postal code in a different comment.